pkg/controller/azure/storage/account/account.go

```diff
@@ -111,7 +113,7 @@ func Add(mgr manager.Manager) error {
 // Reconcile reads that state of the cluster for a Provider acct and makes changes based on the state read
 // and what is in the Provider.Spec
 func (r *Reconciler) Reconcile(request reconcile.Request) (reconcile.Result, error) {
-	log.Printf("reconciling %s: %v", v1alpha1.AccountKindAPIVersion, request)
+	log.V(logging.Debug).Info("reconciling", "kind", v1alpha1.AccountKindAPIVersion, "request", request)
```

Comment:
This is approximation of what a debug message will look like:
```
2019-03-20T00:16:41.846Z        INFO    controller.account.storage.azure.crossplane.io  reconciling        {"kind": "account.storage.azure.crossplane.io/v1alpha1", "request": "crossplane-system/my-test-account"}
```
This is to emphasize the point that perhaps we can improve on reducing redundancy in log messages.

Reply:
I'll open an issue to consider a more streamlined logging name scheme.  It's a good idea, but I don't have a great solution currently.

Reply:
I've opened #388